bluemira/base/reactor.py

```diff
@@ -72,16 +80,12 @@ def save_cad(
         cad_format:
             CAD file format
         """
-        shapes: list[BluemiraGeoT]
-        names: list[str]
-        shapes, names = get_properties_from_components(components, ("shape", "name"))
-        save_cad(shapes, filename, cad_format, names, **kwargs)
 
     @abc.abstractmethod
     def show_cad(
         self,
-        *dims: str,
-        component_filter: Callable[[ComponentT], bool] | None,
+        dim: DIM_3D | DIM_2D,
+        construction_params: ConstructionParams | None = None,
         **kwargs,
```

Comment:
I guess leaving the kwargs here for now in place for the material stuff that would absorb most of the showing options? I have a feeling that the only remaining option would be viewer backend selection

Reply:
#3770